osu.Game/Graphics/UserInterface/BreadcrumbControl.cs

```diff
@@ -15,17 +15,18 @@ namespace osu.Game.Graphics.UserInterface
     public class BreadcrumbControl<T> : OsuTabControl<T>
     {
         private const float padding = 10;
-        private const float item_chevron_size = 10;
+        private readonly float itemChevronSize;
```

Comment:
just make it virtual with a default like we do everywhere else. then override it if/when needed.